defaults/main.yml

```diff
@@ -72,6 +72,14 @@ prometheus_alert_rules:
   annotations:
     description: "{% raw %}{{ $labels.instance }} of job {{ $labels.job }} has Critical CPU load for more than 2 minutes.{% endraw %}"
     summary: "{% raw %}Instance {{ $labels.instance }} - Critical CPU load{% endraw %}"
+- alert: CriticalRAMLoad
+  expr: '(sum(node_memory_MemTotal) - sum(node_memory_MemFree + node_memory_Buffers + node_memory_Cached)) / sum(node_memory_MemTotal) * 100 > 98'
```

Comment:
You probably don't want to `sum()` here, as it will drop the `instance` label.

I would suggest:
```
(1 - ((node_memory_MemFree + node_memory_Buffers + node_memory_Cached) / node_memory_MemTotal)) * 100 > 98
```

Reply:
Yes, this is a good thing